GTFS.DB.PostgreSQL/Collections/PostgreSQLStopTimeCollection.cs

```diff
@@ -263,7 +277,9 @@ public IEnumerable<StopTime> Get()
                         StopHeadsign = reader.ReadStringSafe(),
                         PickupType = (PickupType?)reader.ReadIntSafe(),
                         DropOffType = (DropOffType?)reader.ReadIntSafe(),
-                        ShapeDistTravelled = reader.ReadStringSafe()
+                        ShapeDistTravelled = reader.ReadStringSafe(),
+                        PassengerBoarding = (uint)reader.Read<int>(NpgsqlTypes.NpgsqlDbType.Integer),
```

Comment:
Why aren't you using read int safe? What if these are null? Surely in most cases they will be null? Have you tested/converted any db's yet?

Reply:
done. no way to test it until we import a feed with boarding and alighting to postgres though